### DISCUSSION TITLE
Hi, is there any plan to wrap it as a react component and support typescript?

### DISCUSSION THREAD

**Discussion by ch3nl00ng** [cifkao/html-midi-player#31] [Q&A]
Really love the project, but could not figure out how to use it in React. How can I initialize a html midi player and load midi with Javascript?

Thanks

**Comment by cifkao** [comment]
I don't know much about react, so not sure how much I can help here, but you can check out for example [this page](https://reactjs.org/docs/web-components.html) about the interaction between React and Web Components.

Not sure if you have another question, but you mention TypeScript in the title. `html-midi-player` is written in TypeScript and provides type declarations.
